doc/deployment.md

```diff
@@ -117,6 +123,12 @@ That will make the system look for the repository
 `mediawiki/services/name_in_gerrit` when checking it out in the deploy
 repository.
 
+In case remote name in your deploy repository is different from standard `origin`,
```

Comment:
"The deploy-repo builder script assumes the name of the remote to check out in the deploy repository is `origin`. An alternative name can be configured by invoking (in the source repository):"